.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+* @bosilca
```

Comment:
do we need to get more people here?

Reply:
There are two aspects that need to be considered. First, it is a security risk to allow people to change what is executed on our runners. Second, it remains a critical infrastructure that we want to modify carefully. That being said, this simple means that any code altering this part will need my blessing for now, and as we are getting more familiar with the runners we can increase this list.